src/contracts/pods/EigenPodManager.sol

```diff
@@ -134,68 +137,156 @@ contract EigenPodManager is Initializable, OwnableUpgradeable, Pausable, IEigenP
     /**
      * @notice Deposits/Restakes beacon chain ETH in EigenLayer on behalf of the owner of an EigenPod.
      * @param podOwner The owner of the pod whose balance must be deposited.
-     * @param amount The amount of ETH to 'deposit' (i.e. be credited to the podOwner).
+     * @param amountWei The amount of ETH to 'deposit' (i.e. be credited to the podOwner).
      * @dev Callable only by the podOwner's EigenPod contract.
      */
-    function restakeBeaconChainETH(address podOwner, uint256 amount) external onlyEigenPod(podOwner) {
-        strategyManager.depositBeaconChainETH(podOwner, amount);
-        emit BeaconChainETHDeposited(podOwner, amount);
+    function restakeBeaconChainETH(address podOwner, uint256 amountWei) 
+        external 
+        onlyEigenPod(podOwner) 
+        onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_CREDENTIALS)
```

Comment:
isn't this behavior already enforced at `EigenPod.verifyWithdrawalCredentials`, which is the only path by which this function is called?

Reply:
yup, good point.  In general its probably ok to have redundant pauser modifier on external functions?  wdyt.

For this one, ill fux it since its perm'd to the pod only